script.js

```diff
@@ -0,0 +1,83 @@
+const form = document.querySelector('.form');
+class Book {
+  constructor(title, author, index) {
+    this.title = title;
+    this.author = author;
+    this.index = index;
+  }
+
+  static displayBooks() {
+    const books = Book.getBooks();
+    books.forEach((book) => Book.addBookToList(book));
+  }
```

Comment:
- [ ] it would be better if you use es6 features and use the `arrow function`  instead of `function keyword` that will make you code modern